cymetric/metrics.py

```diff
@@ -441,3 +441,58 @@ def timelist(info):
 
 del _tldeps, _tlschema
 
+# Quantity per GigaWattElectric in Inventory [kg/GWe]
+_invdeps = ['']
+
+_invschema = [
+    ('SimId', ts.UUID),
+    ('AgentId', ts.INT),
+    ('Prototype', ts.STRING),
+    ('Time', ts.INT),
+    ('InventoryName', ts.STRING),
+    ('NucId', ts.INT),
+    ('Quantity', ts.DOUBLE)
+    ]
+
+@metric(name='', depends=_invdeps, schema=_invschema)
+def (inv):
+    """Returns quantity per GWe in the inventory table
+    """
+    inv_index = ['SimId', 'AgentId', 'Prototype', 'Time', 'InventoryName', 'NucId']
+    inv = tools.raw_to_series(expinv,
+                              ['SimId', 'AgentId', 'Prototype', 'Time', 'InventoryName', 'NucId'],
+                              'Quantity')
+    inv = inv.groupby(level=inv_index).sum()
+    inv.name = 'Quantity'
+    rtn = inv.reset_index()
```

Comment:
inside this method, you will need to implement some version of the algorithm you develop to compute the inventory per GWe